sonar-project.properties

```diff
@@ -6,7 +6,7 @@ sonar.tests=src/test/
+sonar.exclusions=src/main/steps/tasklistresponse/proceedings/ProceedingDetailsMapper.ts,src/main/steps/tasklistresponse/proceedings/mainUtils.ts,src/main/steps/tasklistresponse/proceedings/documentUpload/postController.ts,src/main/steps/tasklistresponse/responseCaseSequence.ts,src/main/app/document/DocumentManagementController.ts,src/main/steps/common/confirm-contact-details/checkanswers/ConfirmContactDetailsPostController.ts,src/main/app/controller/GetCaseController.ts,src/main/app/controller/PostController.ts,src/main/app/controller/RespondentSubmitResponseController.ts,src/main/steps/common/confirm-contact-details/checkanswers/ContactDetailsMapper.ts,src/main/steps/common/confirm-contact-details/checkanswers/ConfirmContactDetailsGetController.ts,src/main/steps/tasklistresponse/start/tasklist.ts,src/main/steps/tasklistresponse/tasklistresponseCaseSequence.ts,src/main/app/document/DocumentManagementClient.ts,src/main/steps/respondent/task-list/tasklist.ts,**/__mocks__/**,src/**/*content.ts,src/**/definition.ts,src/**/steps/constants.ts,src/**/case/CaseApi.ts,src/main/steps/respondent/task-list/utils.ts,src/main/app/case/C100CaseApi.ts,src/main/steps/c100-rebuild/check-your-answers/mainUtil.ts,src/main/steps/tasklistresponse/allegations-of-harm-and-violence/check-your-answers/mainUtil.ts,src/main/steps/tasklistresponse/allegations-of-harm-and-violence/SafetyConcernsMapper.ts
```

Comment:
Why there are still exclusions ? Can't we fix all?

Reply:
We are not considering test case for some few files like constants , errorMessages , urls ,etc..